AgreeToTerms/Features/PolicyAgreement/View/PolicyAgreementViewController.swift

```diff
@@ -13,16 +13,35 @@
 import WebKit
 import Combine
 
-class AgreeToTermsViewController: UIViewController {
+/*
+ COMMENTS:
+ - renamed to match feature and make it consistent with view model
+ - provided constructor to inject view model from outside
+ - renamed `subscriptions` to `cancellables` which is sort the naming rule
+ - substituted the mutable force-uncast view attributes with non-mutable instantiated objects
+ */
+
+final class PolicyAgreementViewController: UIViewController {
         
-    private var webView: WKWebView!
-    private var buttonsStackView: UIStackView!
-    private var approveButton: UIButton!
+    private let webView = WKWebView(frame: .zero, configuration: WKWebViewConfiguration())
+    private let buttonsStackView = UIStackView(frame: .zero)
+    private let approveButton = buttonWithText(text: "Approve")
+    private let declineButton = buttonWithText(text: "Decline")
     
     private var finishedLoad: Bool = false
     
-    private var subscriptions = Set<AnyCancellable>()
-    private var policyModel = PolicySignModel()
+    private var cancellables = Set<AnyCancellable>()
+    private let viewModel: PolicyAgreementViewModelProtocol
+
+    init(viewModel: PolicyAgreementViewModelProtocol) {
```

Comment:
For testing, should we inject the protocol to the `viewController` as the `viewModel` ?